### ISSUE TITLE
Send Items Without Auction

### ISSUE THREAD

**Issue by bhgames** [metaplex-foundation/metaplex#51]
It would be nice to be able to send a given item to another wallet without an auction from the UI. Phantom can do this, but it can only do TokenOnlyTransfers. We also support FullRightsTransfers as a protocol, where in addition to the Token, we also do an updateMetadataAccounts call to the token metadata contract where we set the updateAuthority on the metadata to the new wallet as well. This is useful for master editions where you want to confer printing rights to the other wallet going forward, and Phantom simply doesn't do this. Exposing this as an option in the UI and allowing the user to choose via drop down which they would like would be supremely useful. 

As an extra brownie points, I know there are people that would like to do a FullRightsTransfer of their MasterEdition btu would like to for various reasons restrict printing of the new owner. The way to do this is to mint every last printing token in the supply (assuming the MasterEdition does NOT have unlimited supply - if it does, you're screwed here), to your own wallet, and then do the transfer. If you never burn these tokens, the new owner simply cannot mint more. We may wish to expose this as a "FullRightsTransfer with Restrict Printing" option.

I'm aware that FullRightsTransfer for an (Limited/Open) Edition doesnt mean much - they are immutable anyway, but people may still care. It's still nice to have, so I'd like to see it in the UI.

**Comment by gching** [comment]
Makes sense, but my question would be if the `FullRightsTransfer` should be implemented on the wallet side (ie. Phantom) rather than in the Metaplex frontend / UI. Just trying to figure out the separation of concerns here, since in my mind, I feel like the Metaplex frontend / UI should perhaps be solely focused on minting tokens + managing auctions, while the transferring of tokens (`TokenOnlyTransfers` and `FullRightsTransfers`) should be handled by the wallet clients.

**Comment by bhgames** [comment]
I agree, they should, but until they do, which takes time, may as well expose it in the UI :)

**Comment by bhgames** [comment]
Assigning myself since the UI wont let me assign @ysavchenko
